tests/datasets/test_datasets.py

```diff
@@ -468,6 +509,85 @@ def test_empty_dataset():
         _ = dataset.annotations
 
 
+@pytest.mark.parametrize(
+    "atom_proposal_prompt_type, atom_labeling_prompt_type",
+    [("naive_each_step", "per_scene_naive"),
+     ("options_labels_whole_traj", "per_scene_naive"),
+     ("naive_whole_traj", "per_scene_cot"),
+     ("not_a_real_prompt_type", "per_scene_cot"),
+     ("naive_whole_traj", "not_a_real_prompt_type")])
+def test_loading_img_demos(atom_proposal_prompt_type,
+                           atom_labeling_prompt_type):
+    """Test loading a dataset from img demo files.
+
+    NOTE: if you're having issues with this test locally, delete the
```

Comment:
a little sketchy, can we automate this somehow? for example, make the cache directory a flag (if it's not already) and then override it in the test to a temporary directory